src/feasibility_residual.jl

```diff
@@ -46,12 +46,14 @@ end
 
 function residual(nls :: FeasibilityResidual, x :: AbstractVector)
   increment!(nls, :neval_residual)
-  return cons(nls.nlp, x)
+  return cons(nls.nlp, x) - nls.nlp.meta.lcon
 end
```

Comment:
@dpo did we decide that equalities must have `lcon = 0`? I don't think so, right?

Reply:
No, not necessarily.